modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -164,6 +165,8 @@ public PythonClientCodegen() {
         cliOptions.add(new CliOption(DATE_FORMAT, "date format for query parameters")
                 .defaultValue("%Y-%m-%d"));
         cliOptions.add(new CliOption(CodegenConstants.USE_ONEOF_DISCRIMINATOR_LOOKUP, CodegenConstants.USE_ONEOF_DISCRIMINATOR_LOOKUP_DESC).defaultValue("false"));
+        cliOptions.add(new CliOption(USE_PYDANTIC_V2, "Use Pydantic v2 (beta support)")
+                .defaultValue(false));
```

Comment:
Seems like this must be a string: `.defaultValue("false")`

Based on test results: 
```
PythonClientCodegen.java:[169,31] incompatible types: boolean cannot be converted to java.lang.String
```

Reply:
thanks for reporting the issue. fixed via https://github.com/OpenAPITools/openapi-generator/pull/16549/commits/b9cdacc4be088a6578c89e556f1c0d25477cfe9f